.devcontainer/ensure-repos.sh

```
@@ -184,10 +183,22 @@ else
+# to the VS Code plugin. This must be done on the host because the plugin
+# needs this environment variable at server startup.
+ENV_FILE="${HOME}/.phlex-kilo.env"
+: > "${ENV_FILE}"
+
+if [ -f "${HOME}/.config/kilo/kilo.jsonc" ]; then
+  # Replace the baseURL for fnal-litellm specifically.
+  # The range /"fnal-litellm"/,/baseURL/ ensures we only modify the correct provider.
+  # We avoid stripping comments with sed to prevent corrupting URLs (https://),
+  # as the Kilo plugin's JSONC parser handles comments correctly.
+  KILO_CONFIG_CONTENT="$(perl -0777 -pe 's|("fnal-litellm".*?)"baseURL":\s*"[^"]*"|$1"baseURL": "http://host.docker.internal:'${HEADROOM_RELAY_PORT}'"|s; s/"[^"]*"(*SKIP)(*F)|\/\/.*//g; s/^\s+//gm; s/\n//g' "${HOME}/.config/kilo/kilo.jsonc")"
+  echo "KILO_CONFIG_CONTENT='${KILO_CONFIG_CONTENT//'/\\'}'" > "${ENV_FILE}"
```

Shellcheck (0.11.0)

[info] Want to escape a single quote? echo 'This is how it'\''s done'.

(SC1003)